addusers.py

```diff
@@ -137,12 +137,15 @@ def create_user(self, fullname, username, password, description, email, project_
             
             setpass_token = self.setpass.get_token(user.id, password, pin)
             password_url = self.setpass.get_url(setpass_token)
-
-            usr_cfg = dict(config.items('welcome_email'))
-            pwd_cfg = dict(config.items('password_email'))
-
+        
+            usr_cfg = email_defaults.copy() 
```

Comment:
perhaps the email_defaults could be split off to its own class (perhaps a user configuration), this way you could either pass it constructor of the OpenStack class (an OpenStack class has a user configuration).

Another idea would be to take all of the admin user parameters and use the same class to store the admin credentials.